From ecb162d42e2590fafc1cbc9c5fa6eb0fdb47992c Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Tue, 21 Apr 2020 08:06:35 +0200
Subject: [PATCH 1/2] doc: specify that xelatex is used as Latex engine

Building the pdf documentation on readthedocs.org fails with pdflatex.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 doc/conf.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/doc/conf.py b/doc/conf.py
index 0772fb6f0c..b033de9d3a 100644
--- a/doc/conf.py
+++ b/doc/conf.py
@@ -31,6 +31,8 @@ from load_config import loadConfig
 # If your documentation needs a minimal Sphinx version, state it here.
 needs_sphinx = '1.3'
 
+latex_engine = 'xelatex'
+
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-- 
2.26.1

